synapseclient/annotations.py

```diff
@@ -150,9 +150,13 @@ def to_submission_status_annotations(annotations, is_private=True):
     Example: Using this function
         Adding and converting annotations
 
-            from synapseclient.annotations import to_submission_status_annotations, from_submission_status_annotations
+            from synapseclient.client import Synapse
```

Comment:
There's more than one way I could have imported this but I chose to keep with the style of the import below it, rather than `from synapseclient import Synapse`, but let me know if there is a preference.

Reply:
Usually in the docs I've always seen:
```
import synapseclient
syn = synapseclient.Synapse()
syn.login()
```

Since it's used in so many places already lets try to keep this standard.

For other things it's been a bit more ambiguous, like:
```
import synapseclient
from synapseclient import Project, File, Column, Table, EntityViewSchema, EntityViewType
syn = synapseclient.Synapse()
syn.login()
```
Which yeah. Since we're already importing `synapseclient` we could do:
```
synapseclient.Project
synapseclient.File
synapseclient.Column
```

Let's at least keep the standard for now (The top most code block)

Reply:
Noted ✅